src/public/parts/shells/cabinet/public/index.html

```
@@ -0,0 +1,110 @@
+				<form method="dialog"><button class="btn" data-i18n="cabinet.cancel"></button></form>
+				<button id="propSave" type="button" class="btn btn-primary" data-i18n="cabinet.save" data-prop-field></button>
+			</div>
+		</div>
+		<form method="dialog" class="modal-backdrop"><button>close</button></form>
+	</dialog>
+	<dialog id="passwordDialog" class="modal">
+		<div class="modal-box space-y-3 w-11/12 max-w-lg">
+			<h3 class="font-bold text-lg" data-i18n="cabinet.unlockFolder"></h3>
+			<input id="unlockPassword" type="password" class="input input-bordered w-full" />
+			<div class="modal-action">
+				<form method="dialog"><button class="btn" data-i18n="cabinet.cancel"></button></form>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)